components/eam/src/physics/rrtmg/radiation.F90

```diff
@@ -850,7 +850,7 @@ subroutine radiation_tend(state,ptend, pbuf, &
 
     ! Arguments
     logical,  intent(in)    :: is_cmip6_volc    ! true if cmip6 style volcanic file is read otherwise false 
-    real(r8), intent(in)    :: landfrac(pcols)  ! land fraction
+    real(r8), intent(in)    :: landfrac(pcols),dt  ! land fraction
```

Comment:
Why isn't "dt" added on it's own line with a proper comment?

Reply:
thanks. I will fix this as well.